modules/cpmso-feat-auto-update.adoc

```diff
@@ -1,13 +1,50 @@
 // Module included in the following assemblies:
 //
 // * machine_management/control_plane_machine_management/cpmso-managing-machines.adoc
+// * rosa/architecture/control-plane.adoc
+// * osd/architecture/control-plane.adoc
+
+ifeval::["{context}" == "control-plane"]
+:rosa-classic:
+:osd:
+endif::[]
 
 :_mod-docs-content-type: CONCEPT
 [id="cpmso-feat-auto-update_{context}"]
 = Automatic updates to the control plane configuration
+//Not for ROSA/OSD:
+ifndef::openshift-dedicated,openshift-rosa[]
+The `RollingUpdate` update strategy automatically propagates changes to your control plane configuration.
+This update strategy is the default configuration for the control plane machine set.
+
+For clusters that use the `RollingUpdate` update strategy, the Operator creates a replacement control plane machine with the configuration that is specified in the CR.
+
+When the replacement control plane machine is ready, the Operator deletes the control plane machine that is marked for replacement because the machine is either out of spec or in an unhealthy state.
+The replacement machine then joins the control plane.
```

Comment:
This should [stay unchanged](https://docs.openshift.com/container-platform/4.16/machine_management/control_plane_machine_management/cpmso-managing-machines.html#cpmso-feat-auto-update_cpmso-managing-machines) in the self-managed version (single paragraph, same verbiage):
```suggestion
For clusters that use the `RollingUpdate` update strategy, the Operator creates a replacement control plane machine with the configuration that is specified in the CR.
When the replacement control plane machine is ready, the Operator deletes the control plane machine that is marked for replacement.
The replacement machine then joins the control plane.
```